style.css

```diff
@@ -12,41 +12,102 @@ body {
 }
 
 @media (max-width: 992px) {
-  body {
-    margin: 0;
-    padding: 0;
-  }
-
-  #desktop-nav {
-    display: none;
-  }
-
   .ror {
     display: none;
   }
 
-  #mobile-nav {
+  .navbar {
     display: flex;
     flex-direction: row;
     justify-content: space-between;
     padding: 24px;
     background-color: #fff;
   }
 
-  #logo {
-    color: #6070ff;
-    font-size: 18px;
+  .navbar.active {
+    background-color: #6070ff;
+  }
+
+  .nav-branding {
     font-weight: bold;
-    font-style: normal;
+    font-size: 18px;
     line-height: 20px;
+    color: #6070ff;
     text-decoration: none;
+    height: 20px;
   }
 
-  img {
-    width: 18px;
-    height: 12px;
-    justify-content: flex-end;
-    margin: 5px 0 0 0;
+  .nav-link {
+    display: flex;
+    flex-direction: row;
+    justify-content: space-between;
+    list-style-type: none;
+    text-decoration: none;
+    transition: 0.6s ease;
+    padding: 10px 12px;
+    color: #fff;
+    font-weight: 600;
+    font-size: 32px;
+    line-height: 44px;
+  }
+
+  .nav-link:hover {
+    color: dodgerblue;
+    font-weight: 500;
+  }
+
+  .bar {
+    display: block;
+    width: 20px;
+    height: 3px;
+    margin: 4px auto;
+    -webkit-transition: all 0.3s ease-in-out;
+    transition: all 0.3s ease-in-out;
+    background-color: #6070ff;
+    border-radius: 2px;
+  }
+
+  .hamburger {
+    display: block;
+  }
+
+  .hamburger.active .bar:nth-child(2) {
+    opacity: 0;
+  }
+
+  .hamburger.active .bar:nth-child(1) {
+    transform: translateY(6px) rotate(45deg);
+    background-color: #fff;
+  }
+
+  .hamburger.active .bar:nth-child(3) {
+    transform: translateY(-6px) rotate(-45deg);
+    background-color: #fff;
+  }
+
+  .nav-menu {
+    position: fixed;
+    left: -100%;
+    top: 70px;
+    gap: 0;
+    flex-direction: column;
+    background-color: #384ae6;
+    list-style-type: none;
+    width: 100%;
+    height: 100%;
+    opacity: 0.9;
+    text-align: center;
+    transition: 0.3s;
+    margin: 0;
+    padding: 28px;
+  }
```

Comment:
- [ ] Good job so far 👍 It would be great if you could add the blur effect when the mobile menu is opened as shown in the Figma template 💯 
- [ ] Also it would be good to disable the scrolling of the page when the menu is open so that your design will follow the Figma template.
- Your design:
![Annotation 2022-03-08 172637](https://user-images.githubusercontent.com/85113567/157281592-754f3540-9d59-4ca5-a118-2a3b799e7b61.jpg)
- Figma design:
![Annotation 2022-03-08 172745](https://user-images.githubusercontent.com/85113567/157281628-338ebe75-4d68-4e23-9cf6-c84576d8b46b.jpg)

Reply:
OPTIONAL
- Kindly add the right background color, also I noticed the blurred background  is present but because there is no opacity on the background color I cant see this feature kindly make these changes, so your project looks like the original